README.md

```diff
@@ -1,8 +1,34 @@
 # Tekton Tasks Operator
 
-Tekton Tasks Operator is Go operator which takes care about 
-deploying [Kubevirt tekton tasks](https://github.com/kubevirt/kubevirt-tekton-tasks) 
-and example pipelines.
+## Github
+https://github.com/ksimon1/kubevirt-tekton-tasks-operator (WIP)
+
```

Comment:
When should we move the operator to https://github.com/kubevirt/kubevirt-tekton-tasks-operator ?